src/Minify.php

```diff
@@ -154,6 +154,9 @@ protected function load($data)
         if ($this->canImportFile($data)) {
             $data = file_get_contents($data);
 
+            // replace CR linefeeds etc.
+            $data = preg_replace('~\R~', "\n", $data);
```

Comment:
Hm, good question, didn't really think about that I guess :) But of course, you are right, for the sake of performance, str_replace() would be a better choice here... how about

`$data = str_replace(array("\r\n", "\r"), "\n", $data);`

That should cover the WIN and MAC cases - should I file a new PR after I've testet that?